internal/gqlclient/gqlclient.go

```diff
@@ -153,7 +153,7 @@ func (c *Client) RunWithContext(ctx context.Context, request Request, response i
 	name := strutils.Summarize(request.Query(), 25)
```

Comment:
Changes in this file are to facilitate us calling the API without any files to upload.

Because the API was specifically architected around file uploads we can't just use the normal graphql route as the API simply isn't expecting the request to occur in that format.